src/test/java/ladder/domain/ladder/LineTest.java

```diff
@@ -0,0 +1,44 @@
+package ladder.domain.ladder;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import ladder.mock.MockBooleanGenerator;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class LineTest {
+
+    @Test
+    @DisplayName("라인 내 발판들을 생성한다.")
+    void generateRungs() {
+        List<Boolean> rungExist = List.of(true, false, true);
+        Line line = new Line(4, new MockBooleanGenerator(rungExist));
+        List<Rung> rungs = line.getRungs();
+        assertThat(rungs).containsExactly(Rung.EXIST, Rung.EMPTY, Rung.EXIST);
+    }
+
+    @Nested
+    @DisplayName("라인 내 연속된 발판들은 생성되지 않는다.")
```

Comment:
개인적으로 전 Nested를 쓸 땐 계층 단위로 내려가면서 읽히도록 DisplayName을 적습니다
```suggestion
    @DisplayName("라인 내 발판들은")
```
이렇게 해두는 편입니다

Reply:
좋은 것 같아요!. 다른 부분들도 수정했습니다.